integrations/aws/utils.py

```diff
@@ -0,0 +1,256 @@
+import enum
+import json
+from typing import Any, AsyncIterator, Optional
+import typing
+import boto3
+from loguru import logger
+from overrides import AWSPortAppConfig
+from port_ocean.context.event import event
+from port_ocean.context.ocean import ocean
+from starlette.requests import Request
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+from port_ocean.core.handlers.port_app_config.models import ResourceConfig
+
+ACCOUNT_ID_PROPERTY = 'AccountId'
```

Comment:
when we enrich objects with extra properties, we have a convention to add `__` to the property name, so it doesn't conflict with maybe existing property, and so its easier to detect that it was added by the integration